doc/dev/config.rst

```diff
@@ -96,6 +96,8 @@ the configuration. Just like with set_val, you should call apply_changes after
 calling these functions to make sure your changes get applied.
 
 
+.. _dev config defining options:
+_
```

Comment:
This breaks the rendered docs, see https://ceph--64161.org.readthedocs.build/en/64161/dev/config/#changing-configuration-values
```suggestion
```

Reply:
Oh I'm sorry I should have been more explicit. The underscore by itself on line 100 should probably be just an empty line?  

I tried to make a "suggestion" but couldn't see how to make a suggestion for just an empty line.

Reply:
(20 minutes later) Actually, an underscore (the lone underscore on line 100) had to be removed.